js/templates/modals/purchase/complete.html

```diff
@@ -18,7 +18,33 @@
     </div>
     <div class="col2"></div>
   </div>
-  <hr class="clrBr rowHg">
+  <div class="contentBox clrP clrBr padMd">
+    <h5><%= ob.polyT('purchase.completeSection.share.title') %></h5>
+    <p><%= ob.polyT('purchase.completeSection.share.body', { link: "<a class='clrTEm' href='https://openbazaar.org'>https://openbazaar.org</a>" }) %></p>
+    <div class="flex flexKidsExpand gutterH">
+      <% var shareURL = "https%3A%2F%2Fopenbazaar.org" %>
```

Comment:
Is it necessary to escape those slashes?

Reply:
According to the docs it is.